hw/ip/lc_ctrl/lc_ctrl.core

```diff
@@ -17,11 +17,11 @@ filesets:
       - lowrisc:prim:sparse_fsm
       - lowrisc:ip:lc_ctrl_pkg
       - lowrisc:ip:tlul
-      - lowrisc:ip_interfaces:pwrmgr_pkg
+      - lowrisc:virtual_ip:pwrmgr_pkg
       - lowrisc:ip:otp_ctrl_pkg
       - lowrisc:ip:kmac_pkg
       - lowrisc:ip:rv_dm
-      - lowrisc:ip_interfaces:alert_handler_pkg
+      - lowrisc:virtual_ip:alert_handler_pkg
```

Comment:
Another note: The referenced constants probably should just be parameters to all the CIPs, instead of having cross-IP references. (N_ESC_SEV and PING_CNT_DW)

Reply:
These values are used as regular override parameters by prim_esc_receiver, which is instantiated by pwrmgr, lc_ctrl, and rv_core_ibex. It would be really simple to just pass them as regular parameters to these modules. Another PR :)